README.md

```diff
@@ -1,9 +1,19 @@
 <img src="https://www.kubeflow.org/images/logo.svg" width="100">
-Kubeflow is a Cloud Native platform for machine learning based on Google’s internal machine learning pipelines.
+Kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
 
 ---
 Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
```

Comment:
I think we should give this a heading to draw the user's attention.

For example:
```md
---

## Documentation
Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
```

Reply:
Done!